### ISSUE TITLE
Center for Open Science supporting OSS via Gittip as part of Giving Library campaign

### ISSUE THREAD

**Issue by JeffSpies** [gratipay/gratipay.com#1892]
tl;dr: The COS is teaming up with Gittip wants to donate money to OSS. That amount can be increased by as much as $62,000 by sharing this link http://cos.io/givinglibrary and following the instructions ($5 per person!). 

The Center for Open Science (COS; http://cos.io) is a non-profit dedicated to increasing transparency, integrity, and reproducibility in the science; at our core, we're a FOSS (mostly Python-based) dev shop (github.com/CenterForOpenScience) creating technology to facilitate openness (i.e., http://osf.io). Beyond that, we are missioned to support the community at the intersection of open source and open science. 

This month, the Center is releasing a variety of OSS-based projects, and we are making donations to a variety of groups, including PyLadies, PyTN, and, now, OSS developers via Gittip. We have some money ear-marked for donation, but want to increase that amount.

Coincidentally, the Giving Library, a service sponsored by one of our primary funders in order to "help philanthropists give wisely", is offering a pool of $100,000 as part of a social media campaign called Share2Give. For every share on their website (via facebook or twitter), the Giving Library will donate $5 to the non-profit whose page the was shared; the campaign ends on January 31. As of January 9, non-profits have earned $38,000 of the pool, leaving $62,000. 

The COS is going to donate any money earned via the Giving Library campaign to OSS developers via Gittip over the next several years. Here's how it works:

http://cos.io/givinglibrary/

We'd love to be able to earmark as much as $62,000 from this campaign to be donated on Gittip are happy to work with the Gittip community, but need to act rapidly--the pot disappears on the 31st. 

Any ideas would be appreciated!
